http/query_handler.go

```diff
@@ -31,7 +34,9 @@ import (
 )
 
 const (
-	fluxPath = "/api/v2/query"
+	fluxPath           = "/api/v2/query"
+	traceIDHeader      = "Trace-Id"
+	traceSampledHeader = "Trace-Sampled"
```

Comment:
I'm trying to decide if the "trace-sampled"  poses a security risk.  it seems to expose some information about the inner workings of our system.  The traceID is useful besides that (query_log, gray_log, etc), so maybe it's OK if they check jaeger and nothing's found?